test/testenv/deployment.go

```diff
@@ -100,6 +100,16 @@ func (d *Deployment) DeployStandalone(name string) (*enterprisev1.Standalone, er
 	return deployed.(*enterprisev1.Standalone), err
 }
 
+// DeployMonitoringConsole deploys MC instance on specified testenv
+func (d *Deployment) DeployMonitoringConsole(name string, licenseMasterRef string) (*enterprisev1.MonitoringConsole, error) {
```

Comment:
any reason we are including licenseMasterRef here?

Reply:
If we decide to include License Master as part of deployment it would need this Ref. In other scenarios we can just pass empty string.